examples/access-token-server/src/routes/status.ts

```diff
@@ -35,15 +35,16 @@ const router = Router()
  * status endpoint. Returns a status json file provided by devops. Doesnt do
  * anything else as it does not have enough information.
  */
-router.get('/status', async (req, res) => {
+router.get('/status', async (_req, res) => {
   let statusCode = 200
   let status: any = {
     errors: [],
   }
   try {
     const statusString = readFileSync(`${cwd()}/status.json`, 'utf8')
     status = { ...JSON.parse(statusString), ...status }
-  } catch (err) {
+  } catch (err: any) {
```

Comment:
TS 4.4 complains about "Unknown" with leaving this `catch (err)`. We have an `IsErrorLike` test in sdk-rtl we can use for skipping annotating with `: any` but I'm not convinced it's worth it. If it is, it's easy enough to catch this pattern in a follow-up PR and modify it then.

Reply:
I remember @AprilArcus  chimed in on this somewhere but I can't find the conversation

Reply:
It was [here](https://chat.google.com/room/AAAAMqYxkbc/i7Vyiy20YoQ)